websites/G/GameBanana/presence.ts

```diff
@@ -0,0 +1,383 @@
+const presence = new Presence({ clientId: "958520351158050887" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let header: HTMLElement;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "gblogo",
+			startTimestamp: browsingTimestamp
```

Comment:
Not properly formatted, run `yarn lint`

Reply:
Running that didn't change the formatting.

Reply:
Worked fine for me

Reply:
I've ran `yarn lint` multiple times, both with and without `--fix`. The command doesn't change the file at all. Running `yarn lint:ci` tells me "All matched files use Prettier code style!", so I have no idea what I'm doing wrong.